smart-contracts/assembly/contracts/FT/__tests__/WMAS.spec.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  Address,
+  changeCallStack,
+  resetStorage,
+  setDeployContext,
+  mockBalance,
+  mockTransferredCoins,
+} from '@massalabs/massa-as-sdk';
+import { Args, u256ToBytes } from '@massalabs/as-types';
+import { balanceOf, constructor, deposit, withdraw } from '../WMAS';
+import { u256 } from 'as-bignum/assembly';
+
+// address of the contract set in vm-mock. must match with contractAddr of @massalabs/massa-as-sdk/vm-mock/vm.js
+const contractAddr = 'AS12BqZEQ6sByhRLyEuf0YbQmcF2PsDdkNNG1akBJu9XcjZA1eT';
+const user1Address = 'AU12UBnqTHDQALpocVBnkPNy7y5CndUJQTLutaVDDFgMJcq5kQiKq';
+const user2Address = 'AU12BqZEQ6sByhRLyEuf0YbQmcF2PsDdkNNG1akBJu9XcjZA1e8';
+const user3Address = 'AUDeadBeefDeadBeefDeadBeefDeadBeefDeadBeefDeadBOObs';
+
+const TOKEN_NAME = 'TOKEN_NAME';
+const TOKEN_SYMBOL = 'TKN';
+const DECIMALS: u8 = 9;
+const TOTAL_SUPPLY = u256.Zero;
```

Comment:
Useless constants: it's currently only used in the beforeEach function. If you dont plan to reused them elsewhere, you should move the values in that functions.